mayavi/core/engine.py

```diff
@@ -101,6 +101,10 @@ class Engine(HasStrictTraits):
     # Are we running?
     running = Bool(False, record=False)
 
+    # The engine has been closed. This does not stop running the engine.
```

Comment:
Instead you could comment as: "This event is invoked when the engine has been stopped" and don't confuse this with the scenes being closed as that is entirely different and will confuse the reader.